landscape/client/monitor/config.py

```diff
@@ -6,7 +6,7 @@
                "Temperature", "PackageMonitor", "UserMonitor",
                "RebootRequired", "AptPreferences", "NetworkActivity",
                "NetworkDevice", "UpdateManager", "CPUUsage", "SwiftUsage",
-               "CephUsage"]
+               "CephUsage", "ComputerTags"]
```

Comment:
I'm actually on the fence over this. Having things sync automatically for new clients would be nice, but if you add it to `ALL_PLUGINS`, then it becomes enabled by default for any existing client, whether you want it or not. The only way to not have it appear in your workflow would be to set `monitor_plugins=all,the,other,plugins`, which is awkward.

That's probably fine for almost all cases use cases, as client-provided tags are not used a lot in the wild. Clients suddenly messing with my tags after an upgrade could be considered a regression.

Any thoughts on this?

Reply:
I think it should be fine because the clients suddenly messing would just be adding tags if you changed the config file. Nothing would get deleted.